packages/pcd/unknown-pcd/src/UnknownPCD.ts

```diff
@@ -0,0 +1,92 @@
+import { ObjectArgument, PCD, SerializedPCD } from "@pcd/pcd-types";
+
+/**
+ * The globally unique type name of the {@link UnknownPCD}.
+ */
+export const UnknownPCDTypeName = "unknown-pcd";
+
+/**
+ * Interface containing the arguments that 3rd parties use to
+ * initialize this PCD package.
+ */
+export type UnknownPCDInitArgs =
+  | undefined
+  | {
+      /**
+       * Indicates the default behavior for verifying a PCD.  Default
+       * behavior if undefined is to throw an exception since UnknownPCD
+       * doesn't know if the contained PCD is inherently valid or not.
+       */
+      verifyBehavior?: "valid" | "invalid" | "error";
+    };
+
+/**
+ * Defines the essential parameters required for creating a {@link UnknownPCD}.
+ */
+export type UnknownPCDArgs = {
+  /**
+   * The serialized PCD contained in this UnknownPCD.
+   */
+  serializedPCD: ObjectArgument<SerializedPCD>;
+};
+
+/**
+ * Defines the Unknown PCD's claim, which contains the wrapped PCD.
+ */
+export interface UnknownPCDClaim {
+  /**
+   * The serialized PCD wrapped by this UnknownPCD.
+   */
+  serializedPCD: SerializedPCD;
+}
+
+/**
+ * Defines the UnknownPCD proof, which contains private information
+ * about the wrapped PCD.
+ */
+export interface UnknownPCDProof {
+  /**
+   * The deserialization error which lead to the creation of this
+   * UnknownPCD wrapper.
+   */
+  error?: unknown;
+}
```

Comment:
It's pretty arbitrary.  I think I was mostly seeing it as a public/private distinction, or a data/metadata distinction.  There isn't really a proof since there's no verification.